### PR TITLE
Use API GW Cert Handler Image

### PR DESCRIPTION
**Pull request by muralov**
**Description**

Use webhook api-gateway cert-handler image to generate certs for webhooks

Changes proposed in this pull request:

- Configure api-gateway cert-handler image in the job and cronjob
-  Use the generated certificate secret for controller deployment

**Related issue(s)**
[#15346](https://github.com/kyma-project/kyma/issues/15346)

### CODE REVIEW

File: resources/eventing/charts/controller/templates/deployment.yaml [hunk 1]
```diff
@@ -132,7 +132,7 @@ spec:
         - name: cert
           secret:
             defaultMode: 420
-            secretName: eventing-webhook-server-cert
+            secretName: {{ .Values.webhook.secretName }}
```
Comment (raypinto) [inline resources/eventing/charts/controller/templates/deployment.yaml:135]: Let's change this [here](https://github.com/kyma-project/kyma/blob/93184b838bf714cf6965a185cf27be7615706b99/resources/eventing/charts/controller/templates/deployment.yaml#L135) as well!

Comment (raypinto) [reply resources/eventing/charts/controller/templates/deployment.yaml:135]: And delete this file: 
https://github.com/kyma-project/kyma/blob/main/resources/eventing/charts/controller/templates/certmanager.yaml

We need to delete [this](https://github.com/kyma-project/kyma/tree/main/components/eventing-controller/config/certmanager) as well.

File: resources/eventing/charts/controller/templates/webhook-cert-handler.yaml [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  {{- end }}
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ include "controller.fullname" . }}-cert-handler
+rules:
+  - apiGroups: ["apiextensions.k8s.io"]
+    resources: ["customresourcedefinitions"]
+    verbs: ["create", "delete", "get", "patch", "list", "watch", "update"]
```
Comment (raypinto) [inline resources/eventing/charts/controller/templates/webhook-cert-handler.yaml:21]: Does it really need all these roles on "customresourcedefinitions"?

Comment (muralov) [reply resources/eventing/charts/controller/templates/webhook-cert-handler.yaml:21]: Good catch. Create and Delete are not needed as the jobs only get, list, patch, update the CRD.

**Review by raypinto — COMMENTED** [review]
Also delete [this](https://github.com/kyma-project/kyma/blob/93184b838bf714cf6965a185cf27be7615706b99/components/eventing-controller/config/helm/kustomization.yaml#L7) and [this](https://github.com/kyma-project/kyma/blob/main/components/eventing-controller/config/certmanager/certificate.yaml).

Also bump the controller image in the values.yaml

**Review by raypinto — DISMISSED** [review]
Good job!

**Review by raypinto — APPROVED** [review]

**Pull request merged**